src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -252,7 +252,7 @@ const LiveVehicleMarker = ({
   isError,
 }: VehicleIconProps) => {
   const {theme} = useTheme();
-  const fillColor = useTransportationColor(mode, subMode, 'background');
+  const fillColor = useTransportationColor(mode, subMode, false, 'background');
```

Comment:
Shouldn't it be orange if it is an AtB Bestill vehicle?

Reply:
Yes, should have been, some reasons why not:
- It wasn't supported from before
- We don't have support for "See live" on flex travels in the app today
- We don't have enough data in the `TravelDetailsMapScreenComponent` to know if the live vehicle is a flex route or not, and it requires some slightly larger refactoring to work

Reply:
I see. Not worth the extra work for now then.